README.md

```diff
@@ -10,3 +10,9 @@
 $ yarn pack
 $ ./bin/run hello
 ```
+
+## Danger.js integration
+This repository uses Danger.JS to automate some checks in the CI environment (see `.travis.yml`), as well as posting a recap comment on the PR itself.
+Danger requires a github token `DANGER_GITHUB_API_TOKEN` to be setup in the travis settings for this repository.
```

Comment:
Thinking about hbc-techy that works outside travis.
so, if i pass token threw the cli arg and set `DANGER_GITHUB_API_TOKEN ` via node, then danger should work, right? @rbelling

Reply:
yes correct, that token should be available as an environment variable